modules/learn/pages/clusters-and-availability/system-events.adoc

```diff
@@ -7,38 +7,44 @@
-Events are provided by the Cluster Manager, the Data Service, the Index Service, the Search Service, the Eventing Service, the Backup Service, and the Analytics Service.
-Events are also provided to cover security and Views.
+Events are provided for the Cluster Manager, the Data Service, the Index Service, the Search Service, the Eventing Service, the Backup Service, and the Analytics Service.
+Events are also generated for XDCR, security, and Views.
 
-Each event is provided as a JSON document containing a number of key-value pairs: some keys are included in all system events, others are specific to the service that generates the event.
-Each system event that a service generates is of a certain _type_; and this is indicated by an _ID_ and a _description_.
-Informationally, system events are considered to be at different levels of _severity_: supported levels are _info_, _warn_, _error_, and _fatal_.
+Each event is provided as a JSON document containing a number of key-value pairs: some key-value pairs are included in all system events, others are specific to the service for which the event is generated.
+Each key-value pair provides definitive information on the event.
 
-The goal of system-event generation is to provide users with a clear account of what occurs on the cluster: however, system events are not intended to provide detailed, diagnostic information.
+Each system event for a given service is of a certain _type_; and this is indicated by key-value pairs that respectively provide an _ID_ and a _description_.
+Additionally, each event-type is considered to be at a particular level of _severity_: supported levels are _info_, _warn_, _error_, and _fatal_.
+
+The goal of system-event generation is to provide users with a clear account of what occurs on the cluster: however, system events are not intended to provide detailed, diagnostic information (regarding which, see the reference to xref:learn:clusters-and-availability/system-events.adoc#logging[Logging], provided below).
 
 System events are recorded in a _system log_, which is maintained by the Cluster Manager.
-Each node in the cluster has a copy of the system log; and each log is kept up to date by means of _gossip-replication_, between the nodes.
-(See xref:learn:clusters-and-availability/metadata-management.adoc[Metadata Management], for information on gossip-replication.)
+Each node in the cluster has a copy of the system log; and each log is kept up to date by means of _gossip replication_ between the nodes.
+(See xref:learn:clusters-and-availability/metadata-management.adoc[Metadata Management], for information on gossip replication.)
 
-System events can be retrieved from the system log by means of the REST API.
-Two methods of retrieval are supported:
+System events do not need to be explicitly enabled by the administrator: they are collected by default, and can be retrieved by the administrator as required.
```

Comment:
I think we should also include the default number of cluster-wide events we store (10K) and it is possible to configure the value between 3K to 20K. 
The eventLogsLimit can be changed via the internalSettings endpoint as follow: 
 curl -v -X POST http://Administrator:password@127.0.0.1:9000/internalSettings -d eventLogsLimit=3000

The current eventLogsLimit can be retrived via the internalSettings endpoint as follow:

curl -v  http://Administrator:asdasd@127.0.0.1:9000/internalSettings | jq -c .

{"enforceLimits":false,"indexAwareRebalanceDisabled":false,"rebalanceIndexWaitingDisabled":false,"rebalanceIndexPausingDisabled":false,"rebalanceIgnoreViewCompactions":false,"rebalanceMovesPerNode":4,"rebalanceMovesBeforeCompaction":64,"maxParallelIndexers":4,"maxParallelReplicaIndexers":2,"maxBucketCount":30,"eventLogsLimit":3000,"gotraceback":"single","indexAutoFailoverDisabled":true,"certUseSha1":false,"httpNodeAddition":false} 

In a mixed-mode cluster trying to change eventLogsLimit will return an error reading "not supported in mixed version clusters".

Reply:
I have added to this page, and also updated the internalSettings REST page.